framework/src/modules/chain/block_processor_v2.js

```diff
@@ -165,10 +165,19 @@ const validateSchema = ({ block }) => {
 // };
 
 class BlockProcessorV2 extends BaseBlockProcessor {
-	constructor({ blocksModule, bftModule, logger, constants, exceptions }) {
+	constructor({
+		blocksModule,
+		bftModule,
+		dposModule,
+		logger,
+		constants,
+		exceptions,
+	}) {
 		super();
+		const verifyBlockForgerOffset = 2;
```

Comment:
wouldn't it be better if this comes from config?

Reply:
This is a protocol attribute and changing it would require a hard-fork.

Reply:
but rest of the implementation in bft is based on constants.  I don't have a strong opinion on this but maybe @shuse2 can comment.

Reply:
We could define it in `constants.js` but don't expose it as indeed this is hard-forking.

Reply:
It could be in the genesis config, but since it is very specific to v2 and hard to predict what it will affect in the future, let's keep it here or in `constants.js`

Reply:
I will leave it here for now. We can move it to constants when it is used somewhere else.